ember.md

```diff
@@ -357,3 +359,20 @@ Create a build definition for the app that will:
 
 ### 11. Checklist Violations
 `npm test` (in the build definition) will catch errors and reject build
+
+
+
+<a name="addons"></a>
```

Comment:
I don't think this anchor is needed. Markdown links will automatically link to the dasherized version of the heading.